src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const carRentPrice = 40;
+  const rentDiscountForThreeDays = 20;
+  const rentDiscountForSevenDays = 50;
```

Comment:
These variables will be creating every time function is calling. Move them out of function